packages/expo-sms/ios/EXSMS/EXSMSModule.m

```diff
@@ -74,18 +69,12 @@ - (void)messageComposeViewController:(MFMessageComposeViewController *)controlle
     case MessageComposeResultCancelled:
       resolveData = @{@"result": @"cancelled"};
       break;
-
     case MessageComposeResultFailed:
-      rejectMessage = @"SMS message sending failed";
+      rejectMessage = @"User's attempt to save or send an SMS was unsuccessful.";
```

Comment:
what does it mean to save an SMS?

Reply:
it means ... https://developer.apple.com/documentation/messageui/messagecomposeresult/messagecomposeresultfailed?language=objc